src/internal/observable/from.ts

```diff
@@ -34,3 +34,74 @@ export function from<T>(input: ObservableInput<T>, scheduler?: SchedulerLike): O
 
   throw new TypeError((input !== null && typeof input || input) + ' is not observable');
 }
+
+/**
+ * Creates an Observable from an Array, an array-like object, a Promise, an iterable object, or an Observable-like object.
+ *
+ * <span class="informal">Converts almost anything to an Observable.</span>
+ *
+ * ![](from.png)
+ *
+ * `from` converts various other objects and data types into Observables. It also converts a Promise, an array-like, or an
+ * <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols#iterable" target="_blank">iterable</a>
+ * object into an Observable that emits the items in that promise, array, or iterable. A String, in this context, is treated
+ * as an array of characters. Observable-like objects (contains a function named with the ES2015 Symbol for Observable) can also be
+ * converted through this operator.
+ *
+ * ## Examples
+ * ### Converts an array to an Observable
+ * ```javascript
+ * import { from } from 'rxjs/observable/from';
+ *
+ * const array = [10, 20, 30];
+ * const result = from(array);
+ * result.subscribe(x => console.log(x));
+ * // Logs:
+ * // 10 20 30
+ * ```
+ * <a href="https://stackblitz.com/edit/js-tfvhag" target+"_blank">Stackblitz example</a>
```

Comment:
I really appreciate adding stackblitz, but first I think it doesn't really adds value for the small api examples. I think it would be more helpful for the tutorial, we want to add soon. Additionally this messes up the outline in the docs and it isn't conform with accessibility standards. Really thanks for this, but I think this makes sense to add in a seperate pull request